### PR TITLE
Renamed to eos-kalite and include tool manage backups of KA Lite installations

### PR DESCRIPTION
**Pull request by mariospr**
Renamed the source debian package to eos-kalite and add a new package (eos-kalite-tools) to include tools to manage KA Lite installations, which currently include a tool to backup & restore KA Lite installations (eos-kalite-backup).

https://phabricator.endlessm.com/T15449

### CODE REVIEW

File: src/flatpakutils.py [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+
+    logging.info("Searching for {} ref in {}".format(app_id, str(refs)))
+    for refspec in refs.keys():
+        _, remote, ref = OSTree.parse_refspec(refspec)
+
+        if remote_filter and remote != remote_filter:
+            logging.info("{} != {}, skipping...".format(remote, remote_filter))
+            continue
+
+        if str.startswith(ref, 'app/{}'.format(app_id)):
```
Comment (dbnicholson) [inline src/flatpakutils.py]: Probably this is just fine, but one thing I'd suggest after having a second look is to make the test `app/{}/` with the trailing /. That make sure you only catch `com.bar.foo` and not `com.bar.football`.

File: debian/rules [hunk 2]
```diff
@@ -4,4 +4,4 @@
 #export DH_VERBOSE = 1
 
 %:
-	dh $@ --with autoreconf,systemd
```
Comment (dbnicholson) [inline debian/rules]: I don't think you want to remove this (and the Build-Depends). dh-systemd helps to manage the runtime state of systemd units. So, even though dh_systemd_enable is no longer being used to do the installation, dh_systemd_enable and dh_systemd_start will be used to enable/disable and start/restart/stop the units from the maintainer scripts.

If you want to keep the services from being enabled by default, I suggest you just add an override section for dh_systemd_enable:
```
override_dh_systemd_enable:
        dh_systemd_enable --no-enable
```
See also https://github.com/endlessm/eos-boot-helper/blob/master/debian/rules.

**Review by dbnicholson — CHANGES_REQUESTED** [review]

File: src/flatpakutils.py (hunk 1 above)
Comment (mariospr) [reply src/flatpakutils.py]: Makes sense, updating...

File: debian/rules (hunk 2 above)
Comment (mariospr) [reply debian/rules]: Oops, I did not know that. I'll bring dh-systemd back

**Pull request merged**